### PR TITLE
Bulk tagging

### PR DESCRIPTION
**Pull request by schuyler1d**
WIP for bulk tagging and removal.

### CODE REVIEW

File: reviewer/models.py [hunk 1]
```diff
@@ -64,6 +64,40 @@ class Review(models.Model):
...
+        ])
+        return result
+
+    @classmethod
+    def bulk_delete_tag(cls, content_queryset, organization, reviewer, key, decision):
+        """
+        TODO: this may require more work when supporting multi-select
+        """
+        content_type = ContentType.objects.get_for_model(content_queryset.model)
+        result = Review.objects.bulk_create([
```
Comment (schuyler1d) [inline reviewer/models.py]: currently this is just a copy/paste of add, but it needs to be a deletion.
Perhaps we should auto-add a ReviewLog for memory sake?

**Comment by sandramchung** [comment]
Companion to https://github.com/MoveOnOrg/connier/issues/70

File: reviewer/models.py [hunk 2]
```diff
@@ -64,6 +66,58 @@ class Review(models.Model):
...
+        content_type = ContentType.objects.get_for_model(content_queryset.model)
+        obj_ids = [x.id for x in content_queryset]
+        result = Review.objects.filter(
+            organization=organization,
+            reviewer=reviewer,
+            key=key,
+            content_type=content_type,
+            object_id__in=obj_ids
+        ).delete()
+        cls.bulk_clear_review_cache(content_queryset, organization)
```
Comment (schuyler1d) [inline reviewer/models.py]: I think we want to pass the ids into bulk_clear_review_cache. 
1. we already ran the query, so might as well use the obj_ids that we loaded into memory anyway
2. when you run the queryset after the delete, we're depending on a certain transaction race/situation that I don't think it's worth depending on.

Comment (sandramchung) [reply reviewer/models.py]: 👍

**Pull request merged**